src/Vox/Admin/View/index.twig

```diff
@@ -1,43 +1,34 @@
-<!DOCTYPE html>
```

Comment:
Behat tests should be updated to work with new layout or preserve previous key elements.